common/attest_plugin.h

```diff
@@ -15,9 +18,11 @@
 
 #include <openenclave/internal/plugin.h>
 
+OE_EXTERNC_BEGIN
+
 /**
  * Evidence header: the structure that the OE SDK runtime puts on top of
- * evidence data.
+ * evidence data, for format IDs that require it.
```

Comment:
This isn't associated with format IDs any more, consider:
```suggestion
 * evidence data when oe_get_evidence is asked to include the format ID
 * with the evidence.
```

Reply:
Thanks!